examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/Module.scala

```diff
@@ -0,0 +1,65 @@
+package com.example.http4s.blaze.demo.server
+
+import cats.effect.Effect
+import cats.syntax.semigroupk._ // For <+>
+import com.example.http4s.blaze.demo.server.endpoints._
+import com.example.http4s.blaze.demo.server.endpoints.auth.{BasicAuthHttpEndpoint, GitHubHttpEndpoint}
+import com.example.http4s.blaze.demo.server.service.{FileService, GitHubService}
+import fs2.Scheduler
+import org.http4s.HttpService
+import org.http4s.client.Client
+import org.http4s.server.HttpMiddleware
+import org.http4s.server.middleware.{AutoSlash, ChunkAggregator, GZip, Timeout}
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Most of our code is sorting imports lexicographically.

Reply:
Doesn't scalafmt organize the imports on compile? I use NeoVim to write code and I need to fix the `organize imports` command that got broken after an update...

Reply:
It sorts imports within a line, but it does not sort the import lines themselves.  (This operation can, sadly, change semantics in Scala.)